tests/methods/fci-10/input.dat

```diff
@@ -0,0 +1,34 @@
+import forte
+
+refe = -1261.65388210269
+
+molecule {
+2 5
+Fe
+symmetry c1
+}
+
+set globals{
+  basis                  cc-pvdz
+  reference              rohf
+  scf_type               df
+  e_convergence          6
+  d_convergence          5
+  ints_tolerance         0.0
+}
+Escf, wfn = energy('scf', return_wfn=True)
+
+set forte{
+  int_type               df
+  active_space_solver    fci
+  spin_avg_density       true
+  #correlation_solver     sa-mrdsrg
```

Comment:
I think the RDMs have to be computed to detect this bug. The current test actually passes without the fix from this PR.